index.html

```diff
@@ -12034,7 +12068,12 @@ <h4 id="PrimaryGlobal">4.3.6. [Global] and [PrimaryGlobal]</h4>
                 <li>Otherwise, the property exists solely on the interface’s <a class="dfnref" href="#dfn-interface-prototype-object">interface prototype object</a>.</li>
               </ul>
               <p>
-                If the interface has an <a class="dfnref" href="#dfn-iterable-declaration">iterable declaration</a>,
+                If the interface defines an <a class="dfnref" href="#dfn-indexed-property-getter">indexed property getter</a>,
+                then the <span class="estype">Function</span> object is
+                the initial value of the “forEach” data property of <a class="external" href="http://people.mozilla.org/~jorendorff/es6-draft.html#sec-well-known-intrinsic-objects">%ArrayPrototype%</a> (<a href="#ref-ECMA-262">[ECMA-262]</a>, section 6.1.7.4).
+              </p>
+              <p>
+                If the interface has a <a class="dfnref" href="#dfn-pair-iterator">pair iterator</a>,
                 then the <span class="estype">Function</span> <span class="rfc2119">MUST</span>
                 have the same behavior as one that would exist assuming the interface had
                 this <a class="dfnref" href="#dfn-operation">operation</a> instead of the
```

Comment:
`= undefined` is not valid syntax afaict (though this seems to predate this PR).  I thought we were just going to make optional any default to undefined automatically...